src/PotentialCalculation/PotentialCalculationSetup/BoundaryConditions/BoundaryConditionsCylindrical.jl

```diff
@@ -30,13 +30,47 @@ function apply_boundary_conditions_on_r_axis!(  rbpot::AbstractArray{T, 4}, rbi:
     nothing
 end
 
+function apply_boundary_conditions_at_r0!(
+    rbpot::AbstractArray{T, 4}, 
+    rbi::Int,
+    nnz::Int,
+    gw::AbstractArray{T},
+    update_even_points::Val{even_points}
+    ) where {T, even_points}
+    if even_points
+        inz_range = 1:2:nnz # inz odd
+        rbz_range = rbidx.(inz_range)
```

Comment:
Calling `rbidx` on `inz_range` creates a `Vector`, even though this could still be expressed as range (as the name `rbz_range` indicates):
```julia
inz_range = 1:2:nnz
rbz_range = rbidx.(inz_range) # this is a Vector
          = rbidx(inz_range.start):rbidx(inz_range.stop) # this is a StepLinRange
          = 2:rbidx(inz_range.stop) # this is simplified using rbidx(1) == 2
```